bika/lims/browser/aggregatedanalyses/aggregatedanalyses.py

```diff
@@ -99,6 +120,24 @@ def __init__(self, context, request, **kwargs):
                          'state_title',
```

Comment:
Please, remove `cancel` transition. At this state (To be verified), only `verify` and `retract` transitions should be allowed